client/src/core/styles/_typography.scss

```diff
@@ -0,0 +1,157 @@
+//body text
+.typography__body {
+  font-family: $defaultFontFamily;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  font-weight: 400;
+  margin-bottom: 1.5rem;
+}
+
+.typography__body--primary {
+  color: $colourText;
+}
+
+.typography__body--secondary {
+  color: $colourTextSecondary;
+}
+
+.typography__body--strong {
+  color: $colourText;
+  font-family: $fontStrong;
+}
+
+.typography__body--link {
+  color: $colourPrimary;
+  text-decoration: none;
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+//small text
+.typography__small {
+  font-family: $defaultFontFamily;
+  font-size: 0.875rem;
+  line-height: 1.5rem;
+  font-weight: 400;
+  margin-bottom: 1.5rem;
+}
+
+.typography__small--primary {
+  color: $colourText;
+}
+
+.typography__small--secondary {
+  color: $colourTextSecondary;
+}
+
+.typography__small--strong {
+  color: $colourText;
+  font-family: $fontStrong;
+}
+
+.typography__small--link {
+  color: $colourPrimary;
+  text-decoration: none;
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+//serif text
+.typography__serif {
+  font-family: $altFontFamily;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  font-weight: 400;
+  margin-bottom: 1.5rem;
+}
+
+.typography__serif--primary {
+  color: $colourText;
+}
+
+.typography__serif--link {
+  color: $colourPrimary;
+  text-decoration: none;
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+//headings
+.typography__heading {
+  color: $colourText;
+  font-weight: 600;
+  font-family: $fontStrong;
+}
+
+.typography__heading--h1 {
```

Comment:
Should these be mixins applied to the `h` elements in a normalize stylesheet?

Reply:
These should be classes if the idea is we can reuse them on non h tags. while we could use the mixin for that element it means adding another class which seems a bit redundant.